matchzoo/engine/base_preprocessor.py

```diff
@@ -70,6 +70,7 @@ def fit(
     def transform(
         self,
         data_pack: 'mz.DataPack',
+        drop_invalid: bool = False,
```

Comment:
You shouldn't change the signature of `fit` and `transform`. Instead, you should move this parameter to `__init__`, and set the default to `True`, so you don't have to change any of the notebooks.

Reply:
It can not be added to `__init__` since the `drop_invalid` is often set to true for training set but false for validation/testing set.

Reply:
how to handle the NaN  if `drop_invalid` is set to `False`